### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by chriddyp** [plotly/dash-table#644]
Some great issues & suggestions raised in https://community.plot.ly/t/solved-datatables-and-accessibility/31085

**Comment by dgibbon-plotly** [comment]
Comment by a contact

I have just started a new role at Public Health Agency of Canada (PHAC), and would definitely be interested in using Plotly in the future. I really like the design of the Dash callbacks - very simple compared to directly dealing with the events in JS! But, the lack of keyboard accessibility of the various widgets means I can't use them for anything public facing, or even large internal projects. :( Accessibility is highly scrutinized in the government, and it would be quite a lot of work for me to modify the various DCC to make them keyboard accessible. I used to work in procurement accessibility, and it makes a big difference if these things have been at least evaluated according to WCAG 2.1 and/or EN 301 549 2018+, and there is accessibility improvements on the product roadmap.

Highcharts is a good example of keyboard accessibility for visualizations. https://www.highcharts.com/demo/line-ajax

**Comment by Ma-Schw** [comment]
I see the thread was opened 3 years ago, have been changes made to increase accessibility of dash-table? I am also working at a Government Institution (Germany) where accessibility is a big issue. I see a lot of potential in dash apps for our deparment and am currently testing lots of things to see how far I can get with dash.
